com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1072,7 +1081,7 @@ protected override void Render(ScriptableRenderContext renderContext, Camera[] c
             int newCount = m_FrameCount;
             foreach (var c in cameras)
             {
-                if (c.cameraType != CameraType.Preview)
+                if (c != null && c.cameraType != CameraType.Preview)
```

Comment:
Yes, it's because the DeleteLayerRTs() call in the resize callback might delete cameras that were on the render-request list (and create new ones). But now I realize that I don't have to delete any cameras, only need to change their render target, so I just updated teh code. So there is no need for this check anymore.